### PR TITLE
Update context files from owncloud/web

### PR DESCRIPTION
**Pull request by dpakach**
Update the context files from owncloud/web

### CODE REVIEW

File: src/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -103,3 +129,23 @@ Given('user {string} has created the following files', async function (userId, e
...
+    .join('')
+  const toName = table
+    .hashes()
+    .map((data) => data['to-name-parts'])
+    .join('')
+  return webdav.move(user, fromName, toName)
+})
+
+Given(
+  'user {string} has locked file/folder {string} setting following properties',
```
Comment (phil-davis) [inline src/stepDefinitions/filesContext.js:146]:
```suggestion
  'user {string} has locked file/folder {string} setting the following properties',
```

In oC10 core this step has "the" in the text. Can we add that here now? (And adjust the words in web...)

File: src/stepDefinitions/generalContext.js [hunk 2]
```diff
@@ -122,64 +121,45 @@ Then('no message should be displayed on the webUI', function () {
 })
 
 Then(
-  'as {string} the content of {string} should be the same as the local {string}',
-  function (userId, remoteFile, localFile) {
+  'as {string} the content of {string} should be the same as the content of local file {string}',
```
Comment (phil-davis) [inline src/stepDefinitions/generalContext.js:124]: We should adjust these words in oC10 core tests also (core PR needed)

File: src/stepDefinitions/generalContext.js [hunk 3]
```diff
@@ -122,64 +121,45 @@ Then('no message should be displayed on the webUI', function () {
...
+    assertContentOfLocalFileIs(fullPathOfLocalFile, body)
+
+    return this
   }
 )
 
 Then(
-  'as {string} the content of {string} should not be the same as the local {string}',
-  function (userId, remoteFile, localFile) {
+  'as {string} the content of {string} should not be the same as the content of local file {string}',
```
Comment (phil-davis) [inline src/stepDefinitions/generalContext.js:136]: We should adjust these words in oC10 core tests also (core PR needed)

File: src/stepDefinitions/notificationsContext.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ Given('user {string} is sent a notification', function (user) {
     .then((res) => httpHelper.checkStatus(res, 'Could not generate notification.'))
 })
 
-Given('app {string} has been {string}', async function (app, action) {
+Given('app {string} has been {}', async function (app, action) {
```
Comment (phil-davis) [inline src/stepDefinitions/notificationsContext.js:17]: Why is `string` removed? What gets passed?

**Review by individual-it — APPROVED** [review]
need to bump the commit id in the client repo, that will also prove that nothing major is broken

**Review by phil-davis — APPROVED** [review]

**Pull request merged**